pyracing/response_objects/session_data.py

```diff
@@ -19,21 +19,21 @@ def __init__(self, dict):
         self.laps_for_qual_avg = dict['nlapsforqualavg']
         self.laps_for_solo_avg = dict['nlapsforsoloavg']
         self.lead_changes = dict['nleadchanges']
-        self.league_id = dict['leagueid']
-        self.league_season_id = dict['league_season_id']
+        self.league_id = parse_encode(dict['leagueid'])
```

Comment:
I am not super familiar with these, but if they are truly IDs, then parse_encode should have no effect on them. Especially if they come through as ints

Reply:
Agreed. After I included them in the issue that this PR closes, I realized that anything that doesn't have a value comes back from iRacing as an empty string, even if it was originally an int value